packages/web/src/components/comments/CommentActionBar.tsx

```diff
@@ -162,6 +165,12 @@ export const CommentActionBar = ({
     )
   }, [onClickReply, commentId])
 
+  const handleShare = useCallback(() => {
```

Comment:
@schottra heads up we have comment share now, is this something we want to support in entity manager?